trello_service.py

```diff
@@ -0,0 +1,98 @@
+from trello_cards import Item
+import os
+import json
+import requests 
+
+TRELLO_SERVICE_ENDPOINT='http://api.trello.com/1/'
+def get_auth_params():
+    return {    
+                'key': os.environ.get('TRELLO_KEY'), 
+                'token': os.environ.get('TRELLO_TOKEN') 
+            }
+
+def service_url_get_boards():
+    return TRELLO_SERVICE_ENDPOINT+'/members/me/boards'
+
+def service_get_list_from_board(boardID):
+    return TRELLO_SERVICE_ENDPOINT+'/boards/'+boardID+'/lists'
+
+def service_create_card():
+     return TRELLO_SERVICE_ENDPOINT+'/cards'
+
+def service_get_cards_of_list(listId):
```

Comment:
These function names are slightly misleading. Should be `get_cards_of_list_url` etc